nectar/sample-config.toml

```diff
@@ -20,10 +20,12 @@ max_buy_quantity = 0.1
 # If absent, orders are capped by the available base balance.
 max_sell_quantity = 0.1
 
-[maker.maximum_possible_fee]
-# An estimation of the maximum fee that we would expect to pay, used to ensure we always have enough
-# balance to execute an order we publish.
-bitcoin = 0.00009275
+[maker.btc_fee_to_reserve]
+# An estimation of the maximum network fee that we would expect to pay, used to ensure
+# we always have enough balance to execute an order we publish.
+sat_per_vbyte = 34
+# An estimation of the largest Bitcoin transaction we are likely to pay for.
+vbyte_transaction_weight = 850
```

Comment:
Does this setting help with anything?

I am imagining the following scenarios:

1. I am concerned about wasting money on fees and hence I will investigate the config options of nectar to optimize this. In that case, I am ok with stopping to trade in case the network is congested. To configure this, the primary handle is the fee rate.

2. Let's say my configured fee rate is still within the limits of the current network but for some reason, bitcoind has to pull together several inputs to fund an HTLC an hence the transaction is too heavy and would be blocked by this setting.

I don't feel like that is what I want to happen. If my inputs are this scattered then I will have to pay higher fees eventually to use them but this is just how Bitcoin works. Keeping this low is primarily the job of wallets and their coinselection algorithms.

I don't think we should have a config entry for this.

Reply:
What is the alternative and what values would you set then?

The issue is that at this stage, we are not sure of what reasonable values are until we use it in mainnet.